src/ui/components/todomvc-app/component.ts

```diff
@@ -1,24 +1,22 @@
 import Component, { tracked } from "@glimmer/component";
 import Navigo from 'navigo';
 import TodoStore from '../../../utils/todo-store';
-import Todo from '../utils/todo';
+import Todo from '../../../utils/todo';
+import { ENTER } from '../../../utils/keys';
 
 const router = new Navigo(null, true);
 
-
-
 export default class TodoMVCApp extends Component {
-
-  storage: TodoStore;
+  todoStore = new TodoStore();
 
   @tracked todos: Todo[] = [];
```

Comment:
No, this is setting an array per instance (e.g. not on the prototype). Checkout [here](http://www.typescriptlang.org/play/#src=class%20Foo%20%7B%0D%0A%20%20%20%20blah%20%3D%20%5B%5D%0D%0A%7D).

Given:

```js
class Bar { }

class Foo {
    blah: Bar[] = []
}
```

We get the following output:

```js
var Bar = (function () {
    function Bar() {
    }
    return Bar;
}());
var Foo = (function () {
    function Foo() {
        this.blah = [];
    }
    return Foo;
}());
```